src/checkers/cwe_248.mli

```diff
@@ -1,10 +1,27 @@
-(** This module implements a check for CWE-248 (Uncaught Exception)
-An uncaught exception may lead to a crash and subsequentially to other unintended behavior.
-See https://cwe.mitre.org/data/definitions/248.html for detailed description.
+(** This module implements a check for CWE-248: Uncaught Exception.
+
+    An uncaught exception may lead to a crash and subsequentially to other unintended behavior.
```

Comment:
after a crash, the program should be dead. Hence, no other unintended behavior will happen subsequently. 
Ans shouldn't be 'subsequently'?